Userland/DevTools/HackStudio/Debugger/DebuggerObject.h

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Small nitpick: Can we call this something that indicates it has to do with JS integration and not with the debugging mechanism itself, perhaps `DebuggerGlobalJSObject`?

Reply:
Yeah absolutely, good point.